src/utils/extract-resource-url.ts

```diff
@@ -0,0 +1,10 @@
+import { getQueryParam } from "./get-url-param";
+
+// when the glossary dashboard is launched for any activity player activity
+// the resource url needs to be pulled from the embedded query parameter and
+// the api path and the json extension removed from it
+export const extractResourceUrl = (url: string): string => {
```

Comment:
Update this comment to indicate it will likely need to be changed when the activity player support more simple URLs like https://activity-player.concord.org/activity/123

Reply:
I added a note to the readme instead - I thought ti would be more visible there: https://github.com/concord-consortium/glossary-plugin/pull/82/commits/0ed2bc11c76ca6a6b586f2315adf863b4dc73724